### PR TITLE
Feature notifications page

### PR DESCRIPTION
**Pull request by DONOTFORGET-BYTE**

### CODE REVIEW

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 23 min

File: django_project/stakeholder/test_view.py [hunk 1]
```diff
@@ -876,3 +876,229 @@ def test_get_reminder_success(self, mock_serializer, mock_convert_dates):
         response = view.get_reminder(request)
 
         self.assertIsInstance(response, JsonResponse)
+
+
+    # @patch('stakeholder.views.convert_reminder_dates')
```
Comment (dimasciput) [inline django_project/stakeholder/test_view.py:881]: Please remove commented codes

**Review by dimasciput — CHANGES_REQUESTED** [review]

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 23 min

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 22 min

**Review by dimasciput — APPROVED** [review]

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 21 min

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 21 min

**Pull request merged**